app/src/main/res/layout/topic_train_fragment.xml

```diff
@@ -26,35 +27,48 @@
       <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:orientation="vertical"
-        android:padding="32dp">
+        android:orientation="vertical">
+
         <TextView
           android:id="@+id/master_skills_text_view"
           android:layout_width="wrap_content"
           android:layout_height="wrap_content"
+          android:fontFamily="@font/roboto_medium"
```

Comment:
Roboto is included with Android--let's use that rather than including our own TTF files. See: https://stackoverflow.com/a/34445295/3689782.

Reply:
Removed  Font TTF files and used fonts provided in Android.